src/Pickles/Pickles/ObjectModel/Mapper.cs

```diff
@@ -224,13 +226,15 @@ public Example MapToExample(G.Examples examples)
             };
         }
 
-        public ScenarioOutline MapToScenarioOutline(G.ScenarioOutline scenarioOutline)
+        public ScenarioOutline MapToScenarioOutline(G.ScenarioOutline scenarioOutline, string[] tagsToHide)
```

Comment:
It would be more elegant to make the tagsToHide parameter a params array (params string[] tagsToHide). That would remove the need to pass in null arguments on many calls of the method.